genkit-tools/common/src/server/server.ts

```diff
@@ -90,6 +90,12 @@ export function startServer(
     res.end();
   });
 
+  app.post('/api/checkin', bodyParser.json(), async (req, res) => {
```

Comment:
?

Reply:
removed, that was not meant to be here... that was part of runtime discovery experiments.